packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -231,7 +231,20 @@ const main = async () => {
             new program.Option('--inspect', 'enable debugging with --inspect on the node process')
         )
         .addOption(new program.Option('--noHMR', 'disable the client-side hot module replacement'))
-        .action(async ({inspect, noHMR}) => {
+        .addOption(
+            new program.Option(
+                '--babelArgs <babel-args>',
+                'args to pass through to babel'
```

Comment:
```suggestion
                'args to pass through to babel-node'
```
We're using `babel-node`, not the `babel` CLI.